### ISSUE TITLE
Applications of random forest 

### ISSUE THREAD

**Issue by Benfeitas** [NBISweden/workshop_omics_integration#21]
@PayamEmami  has suggested that we could have a session discussing how RF approaches can be applied due to their simplicity and power. I think this is a good idea and think we should discuss it further, it will serve as a good bridge before going to DL.

Pinging @LeandroRitter  and @Ash706

**Comment by LeandroRitter** [comment]
Yes, integration via Random Forest has been our long-standing plan. Unfortunately no time to work on it. Would @PayamEmami be interested in contributing this technique to the course?

**Comment by PayamEmami** [comment]
Thanks. @LeandroRitter @Benfeitas 
Sure I can do that. I think it can be a good compliment to for example neural networks for tabular data.
